tests/test_exceptions.py

```diff
@@ -34,6 +34,7 @@ class TestException(LambdaFWException):
     assert exception.message == "No error message"
     assert exception.error_code == "TEST_ERR"
     assert exception.status_code == 444
+    assert exception.extra == {}
 
 
 def test__lambda_fw_exception__respects_values_provided_during_initialization() -> None:
```

Comment:
As I mentioned [here](https://github.com/pdyba/lambdalizator/pull/101#discussion_r1605810800), you should adjust this test to show the new argument works 😉